### DISCUSSION TITLE
Rounded Corner Picom

### DISCUSSION THREAD

**Discussion by alizteralvord** [owl4ce/dotfiles#177] [Q&A] [answered]
How to disable `picom` rounded corner on maximized window ?

**Comment by owl4ce** [comment, marked as answer]
**Hi @alizteralvord**

## I hope this works as expected.

So, from

https://github.com/owl4ce/dotfiles/blob/bdf9c9d86efc7324f3b8f35ec9d15f2926e39a24/.config/picom.conf#L47-L64

Add a second EWMH property to exclude windows when maximized.

`"_NET_WM_STATE@:32a *= '_NET_WM_STATE_MAXIMIZED_VERT'",`

```conf
# CORNERS
# ---
# Sets the radius of rounded window corners.
# When > 0, the compositor will round the corners of windows.
corner-radius = 8;
# Exclude conditions for rounded corners.
rounded-corners-exclude = [
    "name               *= 'rofi'",
    "name               *= 'screenkey'",
    "name               *= 'tint2'",
    "class_g             = 'Conky'",
    "class_g             = 'GLava'",
    "window_type         = 'dock'",
    "window_type         = 'desktop'",
#   "window_type         = 'tooltip'",
    "_NET_WM_STATE@:32a *= '_NET_WM_STATE_HIDDEN'",
    "_NET_WM_STATE@:32a *= '_NET_WM_STATE_MAXIMIZED_VERT'",
    "_GTK_FRAME_EXTENTS@:c"
];
```

## How can I find these properties?

Use `xprop` and/or `obxprop` (which is openbox-specific).

**Reply by alizteralvord** [reply]
this is what i'm looking for. I've tried searching on Google but can't find the selector
